pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -98,6 +98,9 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         GateOperation::IsingXX,
         GateOperation::IsingYY,
         GateOperation::IsingZZ,
+        GateOperation::SingleExcitation,
+        GateOperation::SingleExcitationMinus,
+        GateOperation::SingleExcitationPlus,
```

Comment:
Do we need the double excitation ops here too? Or is there a single implementation of these?

Reply:
This is due to this fact that less-memory (LS) implementation of ops with more than 2-qubit  hasn't explored in lightning and I'm also not sure even with exploring them we will gain significant performance improvement. For ops like Toffoli, CSWAP and DoubleExcitation*, we rely on precomputed-indices (PI) implementation.

Reply:
Sounds good, thanks Ali